docs/components/zeebe/technical-concepts/internal-processing.md

```diff
@@ -59,8 +59,9 @@ This command can in turn be processed, completing the service task and driving t
 
 When a broker receives a client request, it is written to the **event stream** first, and processed later by the stream processor. If the processing is slow or if there are many client requests in the stream, it might take too long for the processor to start processing the command. If the broker keeps accepting new requests from the client, the backlog increases and the processing latency can grow beyond an acceptable time.
 
-To avoid such problems, Zeebe employs a [backpressure](/self-managed/zeebe-deployment/operations/backpressure.md) mechanism.
-When the broker receives more requests than it can process with an acceptable latency, it rejects some requests.
+To avoid such problems, Zeebe employs some strategies such as [backpressure](/self-managed/zeebe-deployment/operations/backpressure.md) and [flow control](/self-managed/operational-guides/configure-flow-control/configure-flow-control.md).
+In the case of backpressure when the broker receives more requests than it can process with an acceptable latency, it rejects some requests. For flow control it can be used with static write rate limits or throttling which prevents the
+partition from building an excessive backlog of records not exported.
```

Comment:
:wrench: This makes it sound as if flow control and backpressure are different concepts. Flow control has two aspects: limiting write rate and applying backpressure to new user requests.

Reply:
Also corrected this here to the suggested sentence 🙌